langs/tr.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Turkish",
+    "code": "tr",
+    "maintainers": ["enesTufekci"]
```

Comment:
```suggestion
    "maintainers": ["enesTufekci", "helloanil"]
```

Reply:
Hey @enesTufekci, can you also add me to the maintainers list by commiting this suggested change? 💪